1_prep.R

```diff
@@ -5,19 +5,19 @@ source('1_prep/src/munge_nmls.R')
 p1 <- list(
   # pull in files from lake-temperature-model-prep
   # TODO - transfer to Denali using globus
-  # lake - GCM cell crosswalk
-  tar_target(p1_lake_cell_xwalk_csv, '1_prep/in/lake_cell_xwalk.csv', format = 'file'),
-  tar_target(p1_lake_cell_xwalk_df, readr::read_csv(p1_lake_cell_xwalk_csv, col_types=cols())),
   # list of lake-specific attributes for nml modification
   tar_target(p1_nml_list_rds, '1_prep/in/nml_list.rds', format = 'file'),
+  # lake - GCM cell crosswalk, filtered to only those lakes that are
+  # in nml list (able to be modeled by GLM)
+  tar_target(p1_lake_cell_xwalk_csv, '1_prep/in/lake_cell_xwalk.csv', format = 'file'),
+  tar_target(p1_lake_cell_xwalk_df, 
+             readr::read_csv(p1_lake_cell_xwalk_csv, col_types=cols()) %>%
+               filter(site_id %in% names(readr::read_rds(p1_nml_list_rds)))),
```

Comment:
Is there any reason any information other than the names of the list item would could trigger a rebuild of this xwalk when you didn't want it to? If that's a possibility, you may want to pull `names(readr::read_rds(p1_nml_list_rds))` out into its own target.

Reply:
Good point - the nml data could change, but we'd want that to only impact the `p1_nml_objects` target and not this one, so I'll make that change